src/arch/xtensa/configs/apollolake_defconfig

```diff
@@ -1,5 +1,9 @@
 CONFIG_APOLLOLAKE=y
 CONFIG_CAVS_DMIC=y
+CONFIG_CAVS_DMIC_FIR_CUSTOM=y
+CONFIG_CAVS_DMIC_FIR_DECIMATE_BY_2=y
+CONFIG_CAVS_DMIC_FIR_DECIMATE_BY_6=y
+CONFIG_CAVS_DMIC_FIR_DECIMATE_BY_12=y
```

Comment:
What's the advantage of selecting all the decimate options in the defconfig ? Should we jsut use standard values and have the user customise later ?

Reply:
@singalsu I think we should keep here default (full), so all topologies will work. If someone wants to save memory and knows what he is doing he can choose only ones that are needed.

Reply:
Yep I agree, I noticed the added coefficient could fit the build so I don't need to do this. I'll add 12 to standard set.

Reply:
Ok, again please add comments in the Kconfig help so that users know that removing options will save memory at the expense of supported DMIC configs.

Reply:
Yep, I will.